RadixWallet/Features/TransactionReviewFeature/TrackedValidatorStake+Extensions.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+// extension Collection<TrackedValidatorStake> {
```

Comment:
remove dead code?

Reply:
I removed the whole file, i guess it kept the file itself!